scripts/upgrade/v1.0-to-v1.1.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
```

Comment:
If you want to use a seperate script to upgrade it, please import the module. The duplate codes are not a good idea.

Reply:
Done